google-cloud-bigquery-data_transfer/Rakefile

```diff
@@ -0,0 +1,192 @@
+  Dir.glob("test/**/*test.rb")
+    .reject { |file| file.include? "smoke_test" }
+    .each { |file| require_relative file }
+end
+
+namespace :test do
+  desc "Runs tests with coverage."
+  task :coverage do
+    require "simplecov"
+    SimpleCov.start do
+      command_name "google-cloud-bigquery-data_transfer"
+      track_files "lib/**/*.rb"
+      add_filter "test/"
+    end
+
+    Rake::Task["test"].invoke
+  end
+end
+
+desc "Runs the smoke tests."
+task :smoke_test do
+
+  $LOAD_PATH.unshift "lib", "smoke_test"
+  Dir.glob("acceptance/**/*smoke_test.rb").each { |file| require_relative file }
+end
+
+namespace :smoke_test do
+  desc "Runs smoke tests with coverage."
+  task :coverage do
+    require "simplecov"
+    SimpleCov.start do
+      command_name "google-cloud-bigquery-data_transfer"
+      track_files "lib/**/*.rb"
+      add_filter "test/"
+    end
+
+    Rake::Task["smoke_test"].invoke
+  end
+end
+
+# Acceptance tests
+desc "Run the google-cloud-bigquery-data_transfer acceptance tests."
+task :acceptance, :project, :keyfile do |t, args|
+  project = args[:project]
+  project ||=
+    ENV["DATA_TRANSFER_TEST_PROJECT"] ||
+    ENV["GCLOUD_TEST_PROJECT"]
+  keyfile = args[:keyfile]
+  keyfile ||=
+    ENV["DATA_TRANSFER_TEST_KEYFILE"] ||
+    ENV["GCLOUD_TEST_KEYFILE"]
+  if keyfile
+    keyfile = File.read keyfile
+  else
+    keyfile ||=
+      ENV["DATA_TRANSFER_TEST_KEYFILE_JSON"] ||
+      ENV["GCLOUD_TEST_KEYFILE_JSON"]
+  end
+  if project.nil? || keyfile.nil?
+    fail "You must provide a project and keyfile. e.g. rake acceptance[test123, /path/to/keyfile.json] or DATA_TRANSFER_TEST_PROJECT=test123 DATA_TRANSFER_TEST_KEYFILE=/path/to/keyfile.json rake acceptance"
+  end
+  require "google/cloud/bigquery/data_transfer/credentials"
+  (Google::Cloud::Bigquery::DataTransfer::Credentials::PATH_ENV_VARS +
+   Google::Cloud::Bigquery::DataTransfer::Credentials::JSON_ENV_VARS).each do |path|
+    ENV[path] = nil
+  end
+  ENV["DATA_TRANSFER_PROJECT"] = project
+  ENV["DATA_TRANSFER_TEST_PROJECT"] = project
+  ENV["DATA_TRANSFER_KEYFILE_JSON"] = keyfile
+
+  Rake::Task["acceptance:run"].invoke
+end
+
+namespace :acceptance do
+  task :run do
+    Rake::Task["smoke_test"].invoke
+  end
+
+  desc "Run acceptance tests with coverage."
+  task :coverage do
+  end
+
+  desc "Run acceptance cleanup."
+  task :cleanup do
+  end
+end
+
+require "yard"
+require "yard/rake/yardoc_task"
+YARD::Rake::YardocTask.new
+
+desc "Generates JSON output from google-cloud-bigquery-data_transfer .yardoc"
+task :jsondoc => :yard do
+  require "rubygems"
+  require "gcloud/jsondoc"
+
+  registry = YARD::Registry.load! ".yardoc"
+
+  toc_config = {
+    documents: [
+      {
+        type: "toc",
+        title: "Google::Cloud::Bigquery::DataTransfer::V1::DataTypes",
+        modules: [
+          {
+            title: "Google::Cloud::Bigquery::DataTransfer::V1",
+            include: ["google/cloud/bigquery/data_transfer/v1"]
```

Comment:
I believe this path still needs to be `google/cloud/bigquery/datatransfer/v1`, regardless of the camel-case of the module name. This is just what the gcloud-common Angular site app expects. (It has nothing to do with Ruby conventions.)

```sh
$ bundle exec rake jsondoc
...
jsondoc/google/protobuf/timestamp.json
jsondoc/google/protobuf/fieldmask.json
jsondoc/google/cloud.json
jsondoc/google/cloud/bigquery.json
jsondoc/google/cloud/bigquery/datatransfer.json
jsondoc/google/cloud/bigquery/datatransfer/v1.json
jsondoc/google/cloud/bigquery/datatransfer/v1/datatransferserviceclient.json
jsondoc/google/cloud/bigquery/datatransfer/credentials.json
jsondoc/google/cloud/bigquery/datatransfer.json
jsondoc/google/cloud/bigquery/datatransfer/v1.json
jsondoc/google/cloud/bigquery/datatransfer/v1/transferconfig.json
jsondoc/google/cloud/bigquery/datatransfer/v1/transferrun.json

...
```

Reply:
Ah gotcha, updated it.